docker/docker-compose.yml

```diff
@@ -0,0 +1,56 @@
+# This docker-compose file is for deploying a fully working backend
+
+version: '3.7'
+
+services:
+    sqe-database:
+        image: qumranica/sqe-database:0.12.6
+        container_name: SQE_Database
+        environment:
+            ## These must be updated when used in production
+            - MYSQL_ROOT_PASSWORD=none
+            - MYSQL_USER=sqe_user
+            - MYSQL_PASSWORD=mysecretpw
+        expose:
+            - "3306"
```

Comment:
I didn't know we could put these values (and the ports values) in strings.  You learn something new every day.